### ISSUE TITLE
Custom Species Name disappears after switching race, when disabled in the prototype

### ISSUE THREAD

**Issue by DangerRevolution** [Simple-Station/Einstein-Engines#1101]
# Reproduction

Enter the Character UI
Swap race

# Expected behavior

It doesn't do that

# Media

https://github.com/user-attachments/assets/801c1f74-3d3b-4f20-8760-715cf315e688

This happens whether or not you input any text.
This does not return if you restart the round, nor re-enter the lobby. You have to full restart to get the menu back.

**Comment by DangerRevolution** [comment]
Yes, this was also a problem in [dde6130](https://github.com/Simple-Station/Einstein-Engines/commit/dde61304ff56a54844f599758e8b84d95ee9f0fa) (commit after #994). Not sure how this passed review given this is findable within 5 seconds of interacting with the menu.

**Comment by FoxxoTrystan** [comment]
> # Reproduction
> Enter the Character UI Swap race
> 
> # Expected behavior
> It doesn't do that
> 
> # Media
>  2024-10-19.22-25-07.mp4 
> This happens whether or not you input any text. This does not return if you restart the round, nor re-enter the lobby. You have to full restart to get the menu back.

Custom Specie Name menu should be visible to begin with, as goob MRP (if i think this is where its from) did not enable those.

The name should not show up actully unless enabled, did you enable the CustomName variable?

**Comment by DangerRevolution** [comment]
Tested on the latest commit: [d7de808](https://github.com/Simple-Station/Einstein-Engines/commit/d7de8082d3b3c77832a2c190db1e0c03752cb930)

> Custom Specie Name menu should be visible to begin with, as goob MRP (if i think this is where its from) did not enable those.

A) As I said, the screenshot is from the [dde6130](https://github.com/Simple-Station/Einstein-Engines/commit/dde61304ff56a54844f599758e8b84d95ee9f0fa) commit; not the Goob playtest?

> The name should not show up actully unless enabled, did you enable the CustomName variable?

B) The issues are still exactly the same as the video regardless of the prototype:
i) Enabling the prototype still means the custom name field disappears in-between switching
ii) Using the field pre-switch doesn't make the name appear in-game
ii a) Using the name field pre-switch and saving the character slot does not make it appear does not make the name show in-game
ii b) Using the name field pre-switch and changing another variable and saving the character slot does not make the name show in-game
iii) If you type a custom species name into slot A; then switch to slot B then the custom name carries over

# Media

![image](https://github.com/user-attachments/assets/610abc73-3ef4-4832-9219-f2ed92addf9a)
![image](https://github.com/user-attachments/assets/0844e037-40d4-49b2-8e4b-6baa9e2591c2)
Refer to the previous video if you want to see what it looks like

**Comment by DangerRevolution** [comment]
I found the problem; surprisingly the species prototypes aren't validated. I've renamed the issue to be more accurate.

**Comment by Aidenkrz** [comment]
This *should* be fixed now

**Comment by sapphirescript** [comment]
This was fixed in #1120
